src/uploader.c

```diff
@@ -2733,3 +2744,15 @@ STORJ_API storj_upload_state_t *storj_bridge_store_file(storj_env_t *env,
     }
     return state;
 }
+
+STORJ_API void storj_free_uploaded_file_info(storj_file_meta_t *file)
+{
+    if (file) {
+        free((char *)file->id);
```

Comment:
Should we need to check if these are allocated before freeing too?

Reply:
free(NULL) is noop, doesn't? https://stackoverflow.com/a/1938758/2356099